birdbox/birdbox/settings/base.py

```diff
@@ -347,6 +348,9 @@ def path_from_root(*args):
 
 ENGAGE_ROBOTS = config("ENGAGE_ROBOTS", parser=bool, default="False")
 
+# Customise the size of the social-sharing card image
+WAGTAILMETADATA_IMAGE_FILTER = "fill-1200x630"
```

Comment:
I think this gets appended to the filename even when it's less than 1200 x 630 and no resizing occurs. It doesn't actually hurt anything but might make it confusing when the implied dimensions don't match the actual dimensions. Maybe this should be a more generic suffix like "-og" or something?

Reply:
Yeah, that's the unchangeable naming convention of Wagtail. I get your point, but I think the risk of confusion is tolerable.

I'm not a huge fan of how the social sharing images have to get cropped to a letterbox (as per Meta's social-sharing recommendations etc) so we'll have to watch out for people uploading square images that get poorly auto-cropped and people aren't aware until too late